source/_posts/en/tutorial/background.md

```diff
@@ -1,71 +1,93 @@
 ---
-title: 背景
+title: Background
 categories:
 - tutorial
 ---
 
-引言
+Introduction
 -----
 
-2008年，V8 引擎随 Chrome 浏览器横空出世，JavaScript 这门通用的 Web 脚本语言的执行效率得到质的提升。 V8 引擎的出现，注定是 JavaScript 发展史上一个光辉的里程碑。它的出现，让当时研究高性能服务器开发、长时间一筹莫展的 [Ryan Dahl](http://tinyclouds.org/) 有了新的、合适的选择，不久，在2009年的柏林的 JSConf 大会上，基于 JavaScript 的服务端项目 Node.js 正式对外发布。Node.js 的发布，不仅为开发者带来了一个高性能的服务器，还很大程度上推动了前端的工程化，带来了前端的大繁荣。与此同时，因为 JavaScript 执行效率的巨大提升，越来越多的业务逻辑开始在浏览器端实现，前端逻辑越来越重，前端架构随之提上日程。于是，我们谈论的主角，MVVM 模式，走进了 Web 前端的架构设计中。
+Chrome released the V8 JavaScript engine in 2008 and enormously improved the performance of JavaScript. 
 
-概念
+The emergence of the V8 engine is destined to be a significant milestone in the history of JavaScript development.
+
+It provides the best choice for [Ryan Dahl](http://tinyclouds.org/), who want to write a high-performance server and with no progress for a long time.
+
+Soon after, at the JSConf in Berlin in 2009, JavaScript-based server project Node.js officially released.
+
+It gives JavaScript developer a high-performance server and highly promotes workflow tools for Front-End development.
+
+In the meantime, because of the performance improved, more and more business logic migrate from server side to browser.
+
+Browser-side business logic becomes complex, and developer realized that they need a framework to solve these problems.
+
+San, the MVVM JavaScript framework. The architecture solution for web development. 
+
+Concept
 -----
 
-MVVM 模式，顾名思义即 Model-View-ViewModel 模式。它萌芽于2005年微软推出的基于 Windows 的用户界面框架 WPF ，前端最早的 MVVM 框架 [knockout](https://github.com/knockout/knockout) 在2010年发布。
+MVVM(Model-View-ViewModel) pattern. First invented in WPF framework from windows user application framework in 2005. 
 
-一句话总结 Web 前端 MVVM：操作数据，就是操作视图，就是操作 DOM（所以无须操作 DOM ）。
+The earliest MVVM JavaScript framework -- [knockout](https://github.com/knockout/knockout), was released in 2010. 
 
-无须操作 DOM ！借助 MVVM 框架，开发者只需完成包含 **声明绑定** 的视图模板，编写 ViewModel 中业务数据变更逻辑，View 层则完全实现了自动化。这将极大的降低前端应用的操作复杂度、极大提升应用的开发效率。MVVM 最标志性的特性就是 **数据绑定** ，MVVM 的核心理念就是通过 **声明式的数据绑定** 来实现 View 层和其他层的分离。完全解耦 View 层这种理念，也使得 Web 前端的单元测试用例编写变得更容易。
+With MVVM pattern, data are equal to views, and there is no need to write any code to operate the DOM. All the DOM mutations are take over by MVVM frameworks. 
 
-MVVM，说到底还是一种分层架构。它的分层如下：
+JavaScript developer only need to write template including **declare bindings syntax**, the business logic focused on application data in ViewModel. 
 
-- Model: 域模型，用于持久化
-- View: 作为视图模板存在
-- ViewModel: 作为视图的模型，为视图服务
+The template will update automatically when the binding data changed. This design deeply reduces the complexity of the application and increase the efficiency of application development.
```

Comment:
binding data -> bound data
deeply reduces -> significantly
and increase -> while increased